content/citation-tags.tsv

```diff
@@ -276,4 +276,4 @@ Zhu2016_mult_inst_mamm	doi:10.1101/095794
 Zintgraf2017_visualizing	arxiv:1702.04595
 goodfellow2016deep	url:http://www.deeplearningbook.org/
 li2016joint	url:http://www.ijcai.org/Proceedings/16/Papers/403.pdf
-world2004international	url:http://www.who.int/classifications/icd/en/
+world2004international	url:http://www.who.int/classifications/icd/en/
```

Comment:
Can you add back this newline at the end of the file? It's important for the diffs to work properly